.flake8

```diff
@@ -47,3 +68,6 @@ pytest-parametrize-names-type = tuple
 # PT007:
 pytest-parametrize-values-type = tuple
 pytest-parametrize-values-row-type = tuple
+
+# wemake-python-styleguide
+show-source = true
```

Comment:
I'd rather move this above `flake8-pytest-style` settings section so that the native options are sorted separately from external ones.

Reply:
Done.